lib/awful/client.lua

```diff
@@ -8,7 +8,7 @@
 
 -- Grab environment we need
 local gdebug = require("gears.debug")
-local spawn = require("awful.spawn")
+local spawn = nil --TODO v5 deprecate
```

Comment:
Could we pretty-please stop adding more cyclic dependencies to things? :-(

Reply:
Well, that particular one is less evil than usual. It remove the "duty" of spawning things from `awful.client` and concentrate them into `awful.spawn`. Conceptually, I am making things cleaner. But of course, for API compatibility, a temporary hack is required. Where "temporary" means a rather long time.